rest_tools/server/handler.py

```diff
@@ -93,6 +94,10 @@ def initialize(self, debug=False, auth=None, auth_url=None, module_auth_key='',
         self.server_header = server_header
         self.route_stats = route_stats
 
+    @tracing_tools.spanned(kind="server", these=["self.request.method", "self.request.uri"])
```

Comment:
The `uri` could be large, as it is whatever the user entered.  we may not want to include an unbounded user input like this.

Reply:
If we really want to include something, the `self.request.path` would be better.

Reply:
Are you thinking `request.path` is a better identifier?

Reply:
Yes.

Reply:
done